python/lsst/ts/aos/utils/m1m3/M1M3LUT.py

```diff
@@ -69,32 +71,38 @@ async def update_lut_force_balance(
         Rate at which the data is resampled. Default is '1T' (1 minute)
     static_offset: int
         Offset in days to be used when querying the static forces from the EFD.
+    efd_name: str
+        EFD name.
 
     Returns
     -------
     None
     """
+    interval_ms = (end_time - start_time).to(u.ms).value
+    index_type = getattr(M1M3FATable.FAIndex, axis)
+    num_actuators = getattr(M1M3FATable, f"FATABLE_{axis}FA")
```

Comment:
Nice, this is much better than what I had written!

Reply:
Well that needs new criopy, it wasn't in the old one.